eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -244,18 +244,14 @@ jobs:
       # Will run samples described on a file name [service]-samples.txt within the build directory.
       # For example keyvault-samples.txt.
       # The file is written by CMake during configuration when building samples.
-    - task: AzurePowerShell@5
-      displayName: "Run Samples for : ${{ parameters.ServiceDirectory }}"
```

Comment:
You might consider putting the "with AzureCli" text after the service directory because the AZDO UX trims the command line, and IMHO, the service directory parameter is the important part of the text, not the mechanism used for authn.

If AZDO did a competent job of grouping tasks within a job, it would be a different matter, but it doesn't :(.

Reply:
I think I will remove it. It isn't critical info for the end user and self-evident to anyone who reads the task.

Reply:
Given this PR impacts multiple pipelines (which are now all green), and this is a minor fix-up, I don't want to reset CI for it. Will address separately, on top of this.